tests/features/news.feature

```diff
@@ -7,14 +7,15 @@ Feature: News content type.
   Scenario: News information is shown in teasers.
```

Comment:
I think it's time to move this behat into a kernel test. We are slowly doing that for all content type. Steps to do this are:

- Create a NewsRenderTest class, check ProjectRenderTest.php for reference 
- Move news testNews() method from here \Drupal\Tests\oe_theme\Kernel\LegacyContentRenderTest into that class and call it testFullView().
- Create a new method to test teaser in NewsRenderTest and use the new pattern assertions to do that, as we did in ProjectRenderTest, check with @imanoleguskiza if you need help
- Test all cases, such as if there is no label that it falls back to news, etc.
- Remove this behat scenario

Reply:
@ademarco does it mean that I have to remove functional test ContentNewsRenderTest for full News page https://github.com/openeuropa/oe_theme/pull/584/files#diff-01cf51fb99613a3614ce350a61c37c8aR16 ?

Reply:
yeah here we have a bit of legacy to deal with. @yenyasinn I'd say keep it as you are doing it for now, then depending from which one we merge first we remove the kernel test for the full page.